cms/templates/maintenance/_announcement_delete.html

```diff
@@ -0,0 +1,39 @@
+<%page expression_filter="h"/>
+<%inherit file="base.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%block name="title">${_('Delete Announcement')}</%block>
+<%block name="viewtitle">
+<h3 class="info-course">
+    <span>${_('Delete Announcement')}</span>
+</h3>
+</%block>
+
+<%block name="viewcontent">
+<section class="container maintenance-content">
+    <div id="delete-announcement-form" class="studio-view maintenance-form">
+        <div class="container-message wrapper-message">
+          <div class="message has-warnings" aria-hidden="true">
+                <p class="warning">
+                    <span class="icon fa fa-warning"></span>
+                    ${_('Are you sure you want to delete this Announcement?')}
+                </p>
+            </div>
+        </div>
+        <form action="" method="post" class="form-create announcement-container">
+            <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+            <div class="announcement-item">
+                <div class="announcement-content">
+                    ${object.content | n} ## xss-lint: disable=mako-invalid-html-filter
```

Comment:
[Ditto](https://github.com/edx/edx-platform/pull/16496/files#r251327119) for this `xss-lint` comment too.